bitnami/kong/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+name: kong
+version: 1.3.2
```

Comment:
The version should be `0.0.1` for the initial release.

Reply:
Done